src/plot/xy-plot.js

```diff
@@ -289,7 +290,8 @@ class XYPlot extends React.Component {
           className={`rv-xy-plot ${className}`}
           style={{
             width: `${width}px`,
-            height: `${height}px`
+            height: `${height}px`,
+            ...this.props.style
```

Comment:
imo, destructuring this.props doesn't make things more legible, but in the name of consistency I did it.